<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>catch方法</title>
</head>
<body>
    <script>
        const p = new Promise((resolve,reject) => {
            setTimeout(() => {
                //设置p对象的状态为失败，并设置失败值
                reject('出错啦！')
            },1000)
        })

        // p.then(value => {},reason => {
        //     console.log(reason)
        // })

        p.catch(reason => {
            console.warn(reason)
        })
    </script>
</body>
</html>